.github/workflows/test.yml

```diff
@@ -38,9 +38,17 @@ jobs:
           name: debug-files
           path: debug_files.tar.gz
           if-no-files-found: ignore
+
+      - name: Build play-cpp-sdk library
+        run: CXX=clang++-10 CXXFLAGS=-stdlib=libc++ cargo build --package play-cpp-sdk --release
+
+      - name: Build demo project
+        working-directory: demo
+        run: make CXX=clang++-10
```

Comment:
any particular reason why clang 10, and not latest stable (14)?

Reply:
Default clang is 11, headers were missing. I check under `/usr/include/c++` folder, only 9 and 10 is available:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/92778185/184315555-5121acea-71cf-4c38-a233-7a61d4af500e.png">

Reply:
oh ok on ubuntu 20.04